test/replicate.js

```diff
@@ -418,7 +418,7 @@ test('get with { wait: false } returns null if block is not available', async fu
   t.is(await b.get(0), 'a')
 })
 
-test('can disable downloading from a peer', async function (t) {
+test.skip('can disable downloading from a peer', async function (t) {
```

Comment:
Should this still be skipped?

Reply:
Ah, the setDownloading api is still missing

Reply:
(gonna add that in a new PR just to collapse this one)